src/client.ts

```diff
@@ -87,11 +96,20 @@ export class TwilioServerlessApiClient extends events.EventEmitter {
    */
   private client: GotClient;
 
-  constructor(config: ClientConfig) {
+  /**
+   *
+   * @private
+   * @type {Limit}
+   * @memberof TwilioServerlessApiClient
+   */
+  private limit: Limit;
+
+  constructor(config: ClientConfig, concurrency = 50) {
```

Comment:
Can we make this configurable through an environment variable or something? This can be an obscure one.

Reply:
Customers might have other concurrency requirements and I wouldn't want them to have to roll their own library

Reply:
Can definitely make this env var configurable too.